package dev.vanutp.jb_internship_gitea.exceptions

class DirectoryExistsException : ObjectExistsException("A directory with the same name already exists in the index")
